actionview/test/template/translation_helper_test.rb

```diff
@@ -54,6 +54,15 @@ def test_delegates_localize_to_i18n
     end
   end
 
+  def test_returns_missing_tranlation_message_without_span_wrap
+    ActionView::Base.debug_missing_translation = false
```

Comment:
This may cause a test dependency with order. It is better to store the previous value, change it to false and restore to the new value inside the `ensure` block.

```ruby
def
  old_value = ActionView::Base.debug_missing_translation
  ActionView::Base.debug_missing_translation = false

# ....
ensure
  ActionView::Base.debug_missing_translation = old_value
end